opensearch/src/cerbos_opensearch/query.py

```diff
@@ -92,6 +94,13 @@ def traverse_and_map_operands(operand: dict) -> Any:
                 "minimum_should_match": 1,
             }
         }
+    elif operator == "hasIntersection":
+        if not value:
+            value = []
+        values = [value] if isinstance(value, str) else value
+
```

Comment:
Since this is identical to the IN operator, can we move the logic to a common function?